python3/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Python Packages:
 - jmespath (0.9.3)
```

Comment:
Keep CHANGE log for 1.0.0, I already release this tag to dockerhub.

Create new section for `1.1.0` on the top:

## 1.1.0
Python version:
- [3.6.4](https://github.com/docker-library/python/blob/a1aa406bfd8c7b129e6e0ee0ba972b863624ac0d/3.6/jessie/Dockerfile)


Python Packages:
- asn1crypto (0.24.0)
- attrs (17.4.0)
- Automat (0.6.0)
- beautifulsoup4 (4.5.3)
- cffi (1.11.2)
- click (6.7)
- cloudant (2.7.0)
- constantly (15.1.0)
- cryptography (2.1.4)
- cssselect (1.0.3)
- docutils (0.14)
- Flask (0.12)
- gevent (1.2.1)
- greenlet (0.4.12)
- httplib2 (0.10.3)
- ibm-cos-sdk (2.0.0)
- ibm-cos-sdk-core (2.0.0)
- ibm-cos-sdk-s3transfer (2.0.0)
- ibm-db (2.0.8a0)
- idna (2.6)
- incremental (17.5.0)
- itsdangerous (0.24)
- Jinja2 (2.10)
- jmespath (0.9.3)
- kafka-python (1.3.4)
- lxml (3.7.3)
- numpy (1.14.0)
- MarkupSafe (1.0)
- parsel (1.3.1)
- pip (9.0.1)
- pyasn1 (0.4.2)
- pyasn1-modules (0.2.1)
- pycparser (2.18)
- PyDispatcher (2.0.5)
- pyOpenSSL (17.5.0)
- pysolr (3.7.0)
- python-dateutil (2.6.0)
- queuelib (1.4.2)
- requests (2.13.0)
- scipy (1.0.0)
- scikit-learn (0.19.1)
- Scrapy (1.3.3)
- service-identity (17.0.0)
- setuptools (38.4.0)
- simplejson (3.10.0)
- six (1.11.0)
- Twisted (17.1.0)
- virtualenv (15.1.0)
- w3lib (1.18.0)
- watson-developer-cloud (1.0.2)
- Werkzeug (0.14.1)
- wheel (0.30.0)
- zope.interface (4.4.3)